io.github.chcandido.brModelo.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.chcandido.brModelo</id>
+  <launchable type="desktop-id">io.github.chcandido.brModelo.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  <developer_name>Carlos Henrique Candido</developer_name>
+  <name>brModelo</name>
+  <summary>Entity-relationship diagram tool for making conceptual and logical database models.</summary>
+  <description>
+    <p>brModelo is a entity-relationship diagram tool for making conceptual and logical database models.</p>
```

Comment:
It launches in Portuguese(?) for me, and I don't see an option to change it back to English. Unless I missed it, can you include that in the description?

Reply:
@barthalion This software only has the Portuguese language. What do you mean by "that"?

Reply:
Include in the description that the application is only available in portuguese.

Reply:
@guesant did you see this?